lib/util/cli.js

```diff
@@ -53,7 +53,33 @@ const Cli = {
             return null;
         }
     },
-
+    /**
+     * interactive helper to set automation run completion/error note
+     *
+     * @param {TYPE.SupportedMetadataTypes} type note type (error/completion)
+     * @returns {Promise.<string[]>} responses
+     */
+    async updateNotificationEmails(type) {
+        const question1 = {
+            type: 'confirm',
+            name: 'provideEmail',
+            message: `To set run ${type} note, an email address should be set. Do you want to set it?`,
+            default: false,
+        };
+        const question2 = {
+            type: 'input',
+            name: 'updateNotificationNotes',
+            message: `Please enter email addresses separated by a comma:`,
+        };
+        try {
+            if (!(await inquirer.prompt(question1)).provideEmail) {
+                return;
+            }
+            return (await inquirer.prompt(question2)).updateNotificationNotes;
+        } catch (ex) {
+            Util.logger.info(ex);
+        }
+    },
```

Comment:
What kind of other scenarios do you mean? 

In case of an array, not sure how do I exit after asking first question and exiting if the user selects No? If they don't want to set an email address, I don't want to ask for email addresses (2 question)

>>alternatively, you could of course simply require inquirer in whatever class you need these questions and then add this logic there. would allow you to have more on-point questions which will increase UX
But then I would either have to have that in a separate function anyway because I ask these questions twice - if user wants to set errorNote and if they want to set completionNote, but no email addresses were provided

Reply:
actually, this comment i should have deleted after our call :D sorry about that. you already clarified that you needed that exit and I honestly didn't read that into the code the first time around.